docs/plan/Proposal.tex

```diff
@@ -20,23 +20,13 @@
+
+Despite having lost all the mainstream Linux distributions to systemd, enough people continue their resistance to make maintaining several alternative distributions feasible.\footnote{There are those like \textit{Slackware} which has used its BSD-inspired init system before  before \texttt{systemd} entered the stage and continues to do so. Other such examples are \textit{Gentoo} (by default) and \textit{Alpine Linux} which uses \texttt{openRC} and \textit{Void Linux} that adopted \texttt{runit}. There are however even new \textit{systemd-free} distributions which consider this an important feature. Notable xamples are the Debian fork \textit{Devuan} and the Arch Linux fork \textit{Artix}.} However there also exists a minority who still felt oppressed and jumped over to the BSDs to escape the creep of systemd. But this leaves the BSDs in a situation where, if they wanted to innovate their init system and service management, they would have to to try and carefully avoid emoting any PTSD in the refugees they got from Linux. These non-mainstream open source communities are likely too small to further fragment and remain viable\footnote{\textit{FreeBSD} as the by far largest player has seen several failed attempts of getting \texttt{openRC} into the base system. \textit{TrueOS}, a friendly fork that received corporate sponsorship has made completed the switch -- but it died not too long afterwards. \textit{GhostBSD}, a desktop-focused distribution of FreeBSD picked it up and maintained it for years but eventually even migrated back mainly because of the maintenance burden.}. Therefore an unspoken policy of init-system complacency has been adopted (i.e. it's regarded simple, well-known and for a lot use cases in fact \textit{good enough}).
```

Comment:
Two 'before's in a row:
"Which has used its BSD-inspired init system before before..."